yt/visualization/plot_container.py

```diff
@@ -468,6 +470,8 @@ def save(self, name=None, suffix=None, mpl_kwargs=None):
         """
         names = []
         if mpl_kwargs is None: mpl_kwargs = {}
+        if isinstance(name, tuple):
```

Comment:
Maybe change this to `if isinstance(name, (tuple, list))` so it works with lists too. You can leave it as tuple in the docstring, but that way if someone accidentally passes a list it does the right thing.

Reply:
Agreed!